diff -ruN airflow-2.0.2/airflow/www/compile_assets.sh airflow-2.0.2-patch/airflow/www/compile_assets.sh
--- airflow-2.0.2/airflow/www/compile_assets.sh	2021-04-19 22:53:27.000000000 +0200
+++ airflow-2.0.2-patch/airflow/www/compile_assets.sh	2021-04-20 11:00:13.317794832 +0200
@@ -31,4 +31,4 @@
 yarn install --frozen-lockfile
 yarn run build
 
-find package.json yarn.lock static/css static/js -type f | sort | xargs md5sum > "${MD5SUM_FILE}"
+find package.json yarn.lock static/css static/js -type f | sort | xargs md5 -r > "${MD5SUM_FILE}"
diff -ruN airflow-2.0.2/scripts/docker/compile_www_assets.sh airflow-2.0.2-patch/scripts/docker/compile_www_assets.sh
--- airflow-2.0.2/scripts/docker/compile_www_assets.sh	2021-04-19 22:53:27.000000000 +0200
+++ airflow-2.0.2-patch/scripts/docker/compile_www_assets.sh	2021-04-20 11:00:13.337794681 +0200
@@ -38,7 +38,7 @@
     pushd ${www_dir} || exit 1
     yarn install --frozen-lockfile --no-cache
     yarn run prod
-    find package.json yarn.lock static/css static/js -type f | sort | xargs md5sum > "${md5sum_file}"
+    find package.json yarn.lock static/css static/js -type f | sort | xargs md5 -r > "${md5sum_file}"
     rm -rf "${www_dir}/node_modules"
     rm -vf "${www_dir}"/{package.json,yarn.lock,.eslintignore,.eslintrc,.stylelintignore,.stylelintrc,compile_assets.sh,webpack.config.js}
     popd || exit 1
